### PR TITLE
COL-1468 Drop AuthorizedUser model, dev auth against User model

### PR DESCRIPTION
**Pull request by pauline2k**
https://jira.ets.berkeley.edu/jira/browse/COL-1468

### CODE REVIEW

**Comment by pauline2k** [comment]
We also lose the concept of UID, since SuiteC is only interested in user IDs as they come from Canvas or its own tables.

File: squiggy/lib/util.py [hunk 1]
```diff
@@ -31,6 +31,8 @@ def isoformat(value):
 
 
 def to_int(s):
+    if not s:
+        return None
```
Comment (johncrossman) [inline squiggy/lib/util.py]: @pkerschen - If `s` is zero we want to return zero, not None?

**Comment by johncrossman** [comment]
Looks good. Thank you @pkerschen  I put one question.

File: squiggy/lib/util.py (hunk 1 above)
Comment (pauline2k) [reply squiggy/lib/util.py]: @johncrossman Oh, right, thanks. I was trying to catch None input (which throws a TypeError), but that's better handled by catching TypeError below as well as ValueError. I'll amend.

**Pull request merged**